pkg/files/updateRefs.go

```diff
@@ -107,6 +116,11 @@ func UpdateReferences(c *UpdateRefsCommand, dir string) (filesChanged bool, err
 	return filesChanged, nil
 }
 
+func isLineContainsGoImport(line string) bool {
+	// checks if line contains import or github.com
+	return strings.Contains(line, "github.com") && strings.Contains(line, "import")
```

Comment:
In most repos, `import (` is on a different line than the actual imports, e.g. 
```
import (
	"github.com/masterzen/winrm"
	...
)
```
So this logic doesn't catch that.

Reply:
@mdeggies Thanks for your review. 
If I change  condtion to `return strings.Contains(line, "github.com") || strings.Contains(line, "import")` it will solve most of cases.
1. Single line import
2. Multiline import with import starting github.com
But what it won't handle is import without github.com. For example
import (
	"masterzen/winrm"
)
is it a valid case?

If it's the valid case then we need some other approach to handle.